docs/dev-guides/PERSONAL-ENVIRONMENTS.md

```diff
@@ -0,0 +1,523 @@
+# Personal Test Environments
+Personallly provisioned test environments, and the steps to create and manage them, are described here.
+## Background
+### Use Case
+Personal Kubernetes environments allow realistic testing of deployed services without interfering
+with shared environments. This enables faster development cycles, as code can be tested without
+waiting for a release, and without risk of breaking colleagues' environments.
+
+### Where they Live
+A personal environment is contained by a K8s [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
```

Comment:
Nit (maybe incorrect) - is a personal environment contained by a namespace or is it just a namespace?

Reply:
I don't really know. I think of the namespace as the boundary of the environment, like how the EU has borders but isn't the borders. But I'm just guessing.